.golangci-lint-version

```diff
@@ -0,0 +1 @@
+v1.51.2
```

Comment:
This would be the only file we do like this and from the look of it I don't think it is integrated in any way with the action so I'd rather stay away from this (it's also not integrated with dependencies lock).

Reply:
Maybe we could have a `versions.sh` file in `mk/dependencies` ...
The golangci-lint version could be updated in `fmt/ci` (that's how we do golang version update across actions.